StorePhoneAPI/Controllers/CategoryController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using EF_Store.Domain;
+using Microsoft.AspNetCore.Mvc;
+using StorePhone.Сontracts;
+using System.Collections.Generic;
+
+
+namespace StorePhoneAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CategoryController : ControllerBase
+    {
+        private readonly ICategoryService _categoryService;
+
+        public CategoryController(ICategoryService categoryService)
+        {
+            _categoryService = categoryService;
+        }
+
+        [HttpGet("Get")]
```

Comment:
[HttpGet("Get")]
->
[HttpGet]

use just protocol type if mane the same with parameter

get -get
AddCategory - AddCategory
check all actions